assets/js/components/Menu.js

```diff
@@ -95,7 +107,9 @@ const Menu = forwardRef(
 				<ul
 					aria-hidden={ ! menuOpen }
 					aria-orientation="vertical"
-					className="mdc-list"
+					className={ classnames( 'mdc-list', {
+						'mdc-list--non-interactive': nonInteractive,
```

Comment:
Please see **Relevant technical choices** in the PR description for the rationale behind this change.

Reply:
Thanks so much for highlighting these changes in the PR. ❤️ 

It would be nice to have a comment in the code, even one just pointing back to this PR's description, to add so more easy-to-access context. 🙂